Project.toml

```diff
@@ -67,4 +65,4 @@ Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 Unitful = "1986cc42-f94f-5a68-af5c-568840ba703d"
 
 [targets]
-test = ["CSV", "Cairo", "DataFrames", "RDatasets", "Test", "Unitful"]
\ No newline at end of file
+test = ["CSV", "Cairo", "DataFrames", "RDatasets", "Test", "Unitful"]
```

Comment:
As I understand it, for the line_linestyle test script you could have moved StatsBase into `[extras]` and `[targets]`, and the docs will build as long as StatsBase is not in `[deps]`and `[compat]`.  However, I'm ok with removing StatsBase in the line_linestyle testscript, because it simplifies things.